### ISSUE TITLE
Add ability to delete content entry from List page row and 

### ISSUE THREAD

**Issue by westonganger** [locomotivecms/engine#1293]
I see that you can delete via Bulk action checkboxes. However my natural intuition is that there should be a Delete button in the following two places:

- On the List page, on each entries row
- On bottom of the Advanced tab when editing the entry

Note: I came across this need when I accidentally broke the jquery dependency by including `gem 'rails-assets-jquery', '~>3'` in my project. This made the JS stop working at which point I couldnt figure out where the delete buttons were. So I am thinking we should provide a non-JS activated button somewhere.

**Comment by did** [comment]
we'll wait version 4.1 (or upper) for a re-design of the content entry list page. 
Regarding the jQuery issue, I prefer that everything stops working because of a jQuery conflict instead of having a degraded mode which might mask the original problem.

**Comment by westonganger** [comment]
Yeah forget the degraded mode. But simple redesign would help. Sounds good!
